react-training/src/app/components/App/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import logo from '../../../logo.svg';
+import style from './styles.scss';
+import Game from '../../screens/Game';
+
+class App extends Component {
```

Comment:
This is a stateless component, so all this kind of components can be functional components. Check quickly this
https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc